deploy/examples/common-second-cluster.yaml

```
@@ -288,3 +288,57 @@ kind: ServiceAccount
 metadata:
   name: rook-ceph-rgw
   namespace: rook-ceph-secondary # namespace:cluster
+---
+# Allow the ceph mgr to access cluster-wide resources necessary for the mgr modules
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: rook-ceph-mgr-secondary-cluster
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: rook-ceph-mgr-secondary-cluster
+subjects:
+  - kind: ServiceAccount
+    name: rook-ceph-mgr
+    namespace: rook-ceph # namespace:cluster
```

I believe this namespace fix on the binding will be sufficient.
```suggestion
    namespace: rook-ceph-secondary # namespace:cluster
```

---

That fixed it + adjusting the name. Thanks @travisn.